python/cudf/cudf/core/column/string.py

```diff
@@ -1127,6 +1242,28 @@ def sum(self, dtype=None):
     def is_unique(self):
         return len(self.unique()) == len(self)
 
+    @property
+    def is_monotonic_increasing(self):
+        if not hasattr(self, "_is_monotonic_increasing"):
+            if self.nullable and self.has_nulls:
+                self._is_monotonic_increasing = False
+            else:
+                self._is_monotonic_increasing = libcudf.issorted.issorted(
+                    columns=[self]
+                )
+        return self._is_monotonic_increasing
+
+    @property
+    def is_monotonic_decreasing(self):
+        if not hasattr(self, "_is_monotonic_decreasing"):
+            if self.nullable and self.has_nulls:
+                self._is_monotonic_decreasing = False
+            else:
+                self._is_monotonic_decreasing = libcudf.issorted.issorted(
+                    columns=[self], descending=[1]
+                )
+        return self._is_monotonic_decreasing
```

Comment:
I believe both of these were removed to defer down to an implementation in `column.py`

Reply:
bad merge. thanks!